src/inc/msquic.h

```diff
@@ -156,6 +156,12 @@ typedef enum QUIC_SERVER_RESUMPTION_LEVEL {
     QUIC_SERVER_RESUME_AND_ZERORTT,
 } QUIC_SERVER_RESUMPTION_LEVEL;
 
+typedef enum QUIC_SERVER_APP_TICKET_VALIDATION_STATUS {
+    QUIC_SERVER_VALIDATION_DONE,
+    QUIC_SERVER_VALIDATION_PENDING,
+    QUIC_SERVER_VALIDATION_REJECTING,
+} QUIC_SERVER_APP_TICKET_VALIDATION_STATUS;
```

Comment:
As I indicated already, we should just use two booleans instead, but even if we did keep the enum, it shouldn't be public (in msquic.h).